Update repository templates and documents. (#115)
darwinai/covidnet_ui: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request

> about: Suggest an idea for this project

Reviewer: ```suggestion
about: Suggest an idea for this project.
```

title: ''
labels: ''
assignees: ''

---

**Is your feature request related to a problem? Please describe.**

> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

Reviewer: ```suggestion
Please provide a clear and concise description of what the problem is.  For example:  I'm always frustrated when [...]
```

**Describe the solution you'd like**

> A clear and concise description of what you want to happen.

Reviewer: ```suggestion
Please provide a clear and concise description of what you want to happen.
```

**Describe alternatives you've considered**

> A clear and concise description of any alternative solutions or features you've considered.

Reviewer: ```suggestion
Please give us a clear and concise description of any alternative solutions or features you've considered.
```
